src/theory/theory_model_builder.cpp

```diff
@@ -436,6 +526,148 @@ bool TheoryEngineModelBuilder::buildModel(Model* m)
     }
   }
 
+  Trace("model-builder") << "Compute assignable information..." << std::endl;
```

Comment:
The buildModel function is getting really big.  Can we break this new code out into a separate function?